common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/extensions/ErrataSortExtension.java

```diff
@@ -0,0 +1,66 @@
+package com.redhat.cloud.notifications.qute.templates.extensions;
+
+import io.quarkus.logging.Log;
+import io.quarkus.qute.TemplateExtension;
+import java.util.List;
+import java.util.Map;
+
+public class ErrataSortExtension {
```

Comment:
Did you create the extension to sort the data only once when the template is rendered, instead of sorting the data in the aggregator every time a new payload is aggregated?